java/client/src/test/java/glide/api/models/ClusterTransactionTests.java

```diff
@@ -42,6 +43,9 @@ public void transaction_builds_protobuf_request() {
                                 .addArgs(RETURN_OLD_VALUE)
                                 .build()));
 
+        transaction.del(new String[] {"key1", "key2"});
```

Comment:
Why not add these tests to test utilities instead?

Reply:
That's for IT tests. This is for UT tests.

Reply:
See: https://github.com/Bit-Quill/glide-for-redis/pull/91